man/io_uring_enable_rings.3

```diff
@@ -0,0 +1,40 @@
+.\" Copyright (C) 2023 nick black <dankamongmen@gmail.com>
+.\"
+.\" SPDX-License-Identifier: LGPL-2.0-or-later
+.\"
+.TH io_uring_enable_rings 3 "November 15, 2021" "liburing-2.1" "liburing Manual"
```

Comment:
This should be the current date, and liburing-2.4 as that will be the next release.